[2017-08-18 09:12:00] SD.ERROR: WORKER Error Redis::connect(): connect() failed: Connection timed out (/app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:408) [] []
[2017-08-18 09:12:01] SD.ERROR: WORKER EXIT UNEXPECTED Uncaught RedisException: Redis server went away in /app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:409 Stack trace: #0 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php(409): Redis->getLastError() #1 /app/sd/vendor/tmtbe/swooledistributed/src/Server/SwooleDistributedServer.php(122): Server\Asyn\Redis\RedisAsynPool->getSync() #2 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Start.php(356): Server\SwooleDistributedServer->start() #3 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Start.php(70): Server\Start::startSwoole() #4 /app/sd/bin/start_swoole_server.php(11): Server\Start::run() #5 {main}   thrown (/app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:409) Stack trace: #0 unknown(0): app\AppServer->checkErrors()  [] []
[2017-08-18 09:14:30] SD.ERROR: WORKER Error Redis::connect(): connect() failed: Connection timed out (/app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:408) [] []
[2017-08-18 09:14:30] SD.ERROR: WORKER EXIT UNEXPECTED Uncaught RedisException: Redis server went away in /app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:409 Stack trace: #0 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php(409): Redis->getLastError() #1 /app/sd/vendor/tmtbe/swooledistributed/src/Server/SwooleDistributedServer.php(122): Server\Asyn\Redis\RedisAsynPool->getSync() #2 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Start.php(356): Server\SwooleDistributedServer->start() #3 /app/sd/vendor/tmtbe/swooledistributed/src/Server/Start.php(70): Server\Start::startSwoole() #4 /app/sd/bin/start_swoole_server.php(11): Server\Start::run() #5 {main}   thrown (/app/sd/vendor/tmtbe/swooledistributed/src/Server/Asyn/Redis/RedisAsynPool.php:409) Stack trace: #0 unknown(0): app\AppServer->checkErrors()  [] []
